M2/Macaulay2/d/convertr.d

```diff
@@ -298,7 +298,7 @@ export convert0(e:ParseTree):Code := (
 	      is t:Token do Code(augmentedAssignmentCode(
 		      b.Operator.entry, convert(b.lhs), convert(b.rhs), t.entry, pos))
 	      else Code(augmentedAssignmentCode(
-		      b.Operator.entry, dummyCode, dummyCode, dummySymbol, dummyPosition)) -- CHECK
+		      b.Operator.entry, dummyCode, dummyCode, dummySymbol, pos)) -- CHECK
```

Comment:
Maybe also remove the `CHECK` while you're editing this line?

Reply:
Also, are the `dummyCode`s and `dummySymbol` right?

Reply:
I think if we end up at this line, we're bound to get a "not implemented" error, so I don't think it really matters.  We might as well call `convert` on the two sides, I suppose.

I think `dummySymbol` probably makes the most sense for `info` in this case.  This field is kind of a hack -- it's either the variable name  we're assigning to or the operator on the left hand side, depending on the situation.